### ISSUE TITLE
Fieldset Toggleable default is true, and cannot be changed

### ISSUE THREAD

**Issue by rhornrrs** [primefaces/primeng#4737]
### There is no guarantee in receiving a response in GitHub Issue Tracker, If you'd like to secure our response, you may consider *PrimeNG PRO Support* where support is provided within 4 business hours

**I'm submitting a ...**  (check one with "x")
```
[X] bug report => Search github for a similar issue or PR before submitting
[ ] feature request => Please check if request is not on the roadmap already https://github.com/primefaces/primeng/wiki/Roadmap
[ ] support request => Please do not submit support request here, instead see http://forum.primefaces.org/viewforum.php?f=35
```

**Current behavior**
create a p-fieldset or p-fieldset [toggleable]="false" and the fieldset is still toggleable

**Expected behavior**
that when the "toggleable" attribute is not defined, or when set to false, the fieldset will not be toggleable

**Minimal reproduction of the problem with instructions**
 create a field set with out the toggleable property, or with it set to false

**What is the motivation / use case for changing the behavior?**
The current behavior is not correct, the toggleable flag is being ignored

**Please tell us about your environment:**
Windows 8
Eclipse Mars
Jboss 5.x

* **Angular version:** 2.0.X
5.1.2

* **PrimeNG version:** 2.0.X
5.0.2

* **Browser:** [all | Chrome XX | Firefox XX | IE XX | Safari XX | Mobile Chrome XX | Android X.X Web Browser | iOS XX Safari | iOS XX UIWebView | iOS XX WKWebView ]
Chrome Version 63.0.3239.108
FireFox 57.0.2
IE 11 version 11.0.9600.18861 Update version 11.0.49

 
* **Language:** [all | TypeScript X.X | ES6/7 | ES5]
ES5

* **Node (for AoT issues):** `node --version` =   
v 6.10.0

**Comment by Menecats** [comment]
+1

**Comment by solenski** [comment]
+1

**Comment by Menecats** [comment]
Fixed in PR #4808

**Comment by cagataycivici** [comment]
4527
